sass/components/_status-icon.scss

```diff
@@ -193,3 +193,18 @@
         top: -1px;
     }
 }
+
+.status-dropdown-menu {
+    .online--icon > svg {
+        fill: rgba(var(--online-indicator-rgb), 1);
```

Comment:
When I tried to use `rgb()` here, I was getting a compile error stating that the `$green` argument was missing. Using `rgba()` for now until I figure out why. 🤔